src/style.css

```diff
@@ -1,3 +1,123 @@
+* {
+  box-sizing: border-box;
+}
+
+p,
+h3 {
+  margin: 0;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
```

Comment:
Create file `card.scss` and move styles there
![image](https://github.com/mate-academy/layout_product-cards/assets/70009205/7a36a529-ae7e-4559-b900-6c6d7283edac)